bundler/lib/bundler/templates/newgem/newgem.gemspec.tt

```diff
@@ -21,9 +21,9 @@ Gem::Specification.new do |spec|
 
   spec.metadata["allowed_push_host"] = "TODO: Set to your gem server 'https://example.com'"
 
-  spec.metadata["homepage_uri"] = spec.homepage
-  spec.metadata["source_code_uri"] = "TODO: Put your gem's public repo URL here."
-  spec.metadata["changelog_uri"] = "TODO: Put your gem's CHANGELOG.md URL here."
+  spec.metadata["homepage_uri"] = "<%= config[:homepage_uri] %>"
```

Comment:
can you add this change instead to line 13 and revert this line to how it was. There is duplication in the spec that we are trying to wrangle.